sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -36,6 +36,20 @@ static pi_result QueueRelease(pi_queue Queue, pi_queue LockedQueue);
 
 namespace {
 
+// The following supports an experimental feature to import host ptrs
+// into USM, improving data transfer performance.
+// Environment variable control over the feature.
+static const bool USMHostPtrImport = [] {
+  const char *USMHostPtrImportStr = std::getenv("SYCL_USM_HOSTPTR_IMPORT");
```

Comment:
this needs to be documented in https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md